base/cvd/cuttlefish/common/libs/utils/files.h

```diff
@@ -41,6 +41,9 @@ Result<void> CreateSymLink(const std::string& target, const std::string& link,
 bool FileHasContent(const std::string& path);
 Result<std::vector<std::string>> DirectoryContents(const std::string& path);
 bool DirectoryExists(const std::string& path, bool follow_symlinks = true);
+inline bool IsDirectory(const std::string& path) {
+  return DirectoryExists(path);
+};
```

Comment:
Is this new function necessary in this PR? I don't see it being used anywhere.

Reply:
Discussed that with Jorge and he seems to be fine with this. The alternative is to add a comment to claim these two are the same.